### ISSUE TITLE
[cmake] Make RELEASE_NOTES docs (Trac #1048)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1042]
Use proper rst for the RELEASE_NOTES in a doc that's linked into the meta-project docs.  We'll need to figure out how to separate offline notes from sim, icerec, etc...but that shouldn't be too tough.

Migrated from https://code.icecube.wisc.edu/ticket/1048
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:10", 
    "description": "Use proper rst for the RELEASE_NOTES in a doc that's linked into the meta-project docs.  We'll need to figure out how to separate offline notes from sim, icerec, etc...but that shouldn't be too tough.", 
    "reporter": "olivas", 
    "cc": "nega, david.schultz@icecube.wisc.edu", 
    "resolution": "fixed", 
    "_ts": "1458335650323600", 
    "component": "cmake", 
    "summary": "[cmake] Make RELEASE_NOTES docs", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-07-14T18:19:21", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150722T15:25:47: 
* david.schultz commented:

> see r2174/IceTray for an example of project release notes. thoughts?


* david.schultz changed cc from "nega" to "nega, david.schultz@icecube.wisc.edu"

**Comment by icecube-trac** [comment]
Trac update at 20150722T16:44:24: 
* david.schultz commented:

> steal ticket from olivas


* david.schultz changed owner from "olivas" to "david.schultz"
* david.schultz changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150724T17:42:59: david.schultz commented:

> complete for offline-software.  simulation and icerec still to do.

**Comment by icecube-trac** [comment]
Trac update at 20150806T22:29:57: david.schultz commented:

> complete for simulation

**Comment by icecube-trac** [comment]
Trac update at 20150806T22:50:22: 
* david.schultz commented:

> Will do the icerec projects later.  milestone:"IceProd 2.0" is looming.


* david.schultz changed milestone from "offline V15-07-00" to "offline V15-09-00"

**Comment by icecube-trac** [comment]
Trac update at 20151125T16:38:16: 
* david.schultz commented:

> In changeset 139850:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="139850"
> get the RELEASE_NOTES into the docs. fix any other sphinx errors. closes #1042
> ```


* david.schultz changed status from "accepted" to "closed"
* david.schultz changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:10: 
* olivas changed milestone from "offline V15-12-00" to ""
* olivas commented:

> Milestone offline V15-12-00 deleted
